src/pear/config.vsh.yaml

```diff
@@ -0,0 +1,145 @@
+            Specify the type of statistical test. Two options are available. 1: Given the minimum allowed overlap, test using the highest OES. Note that due to its discrete nature, this test usually yields a lower p-value for the assembled read than the cut- off (specified by -p). For example, setting the cut-off to 0.05 using this test, the assembled reads might have an actual p-value of 0.02.
+            2. Use the acceptance probability (m.a.p). This test methods computes the same probability as test method 1. However, it assumes that the minimal overlap is the observed overlap with the  highest OES, instead of the one specified by -v. Therefore, this is not a valid statistical test and the 'p-value' is  in fact the maximal probability for accepting the assembly. Nevertheless, we observed in practice that for the case the actual overlap sizes are relatively small, test 2 can correctly assemble more reads with only slightly higher false-positive rate.
+          required: false
+          default: 1
+        - name: --emperical_freqs
+          alternatives: -e
+          type: boolean_true
+          description: |
+            Disable  empirical base frequencies.
+        - name: --score_method
+          alternatives: -s
+          type: integer
+          description: |
+            Specify the scoring method. 1. OES with +1 for match and -1 for mismatch. 2: Assembly score (AS). Use +1 for match and -1 for mismatch multiplied by base quality scores. 3: Ignore quality scores and use +1 for a match and -1 for a mismatch.
+          required: false
+          default: 2
+        - name: --phred_base
+          alternatives: -b
+          type: integer
+          description: |
+            Base PHRED quality score.
+          required: false
+          default: 33
+        - name: --memory
+          alternatives: -y
+          type: string
+          description: |
+            Specify the amount of memory to be used. The number may be followed by one of the letters K, M, or G denoting Kilobytes, Megabytes and Gigabytes, respectively. Bytes are assumed in case no letter is specified.
+          required: false
+          example: 4G
+        - name: --cap
+          alternatives: -c
+          type: integer
+          description: |
+            Specify  the upper bound for the resulting quality score. If set to zero, capping is disabled.
+          required: false
+          default: 40
+        - name: --nbase
+          alternatives: -z
+          type: boolean_true
+          description: |
+            When merging a base-pair that consists of two non-equal bases out of which none is degenerate, set the merged base to N and use the highest quality score of the two bases
+  resources:
+    - type: bash_script
+      path: script.sh
+  test_resources:
+    - type: bash_script
+      path: test.sh
+    - type: file
+      path: test_data
+platforms:
+  - type: docker
+    image: quay.io/biocontainers/pear:0.9.6--h9d449c0_10
```

Comment:
Please add a command that writes something like `pear: "0.9.6"` to `/var/software_versions.txt`. You can base this command on the [nf-core version command](https://github.com/nf-core/modules/blob/master/modules/nf-core/pear/main.nf#L39-L42)